manta-accounting/src/wallet/signer/mod.rs

```diff
@@ -106,6 +122,171 @@ where
     ) -> LocalBoxFutureResult<SignWithTransactionDataResult<C>, Self::Error>
     where
         TransferPost<C>: Clone;
+
+    /// Returns the transfer [`Parameters`] corresponding to `self`.
+    fn transfer_parameters(&mut self) -> LocalBoxFutureResult<Parameters<C>, Self::Error>;
+}
+
+/// Signer Initial Synchronization Data
+#[cfg_attr(
+    feature = "serde",
+    derive(Deserialize, Serialize),
+    serde(
+        bound(
+            deserialize = r"
+                Utxo<C>: Deserialize<'de>,
+                UtxoAccumulatorWitness<C>: Deserialize<'de>,
+                Nullifier<C>: Deserialize<'de>,
+            ",
+            serialize = r"
+                Utxo<C>: Serialize,
+                UtxoAccumulatorWitness<C>: Serialize,
+                Nullifier<C>: Serialize,
+            ",
+        ),
+        crate = "manta_util::serde",
+        deny_unknown_fields
+    )
+)]
+#[derive(derivative::Derivative)]
+#[derivative(
+    Clone(bound = "Utxo<C>: Clone, UtxoAccumulatorWitness<C>: Clone, Nullifier<C>: Clone"),
+    Debug(bound = "Utxo<C>: Debug, UtxoAccumulatorWitness<C>: Debug, Nullifier<C>: Debug"),
+    Eq(bound = "Utxo<C>: Eq, UtxoAccumulatorWitness<C>: Eq, Nullifier<C>: Eq"),
+    Hash(bound = "Utxo<C>: Hash, UtxoAccumulatorWitness<C>: Hash, Nullifier<C>: Hash"),
+    PartialEq(
+        bound = "Utxo<C>: PartialEq, UtxoAccumulatorWitness<C>: PartialEq, Nullifier<C>: PartialEq"
+    )
+)]
+pub struct InitialSyncData<C>
+where
+    C: transfer::Configuration + ?Sized,
+{
+    /// UTXO Data
+    pub utxo_data: Vec<Utxo<C>>,
+
+    /// Membership Proof Data
+    pub membership_proof_data: Vec<UtxoAccumulatorWitness<C>>,
```

Comment:
Maybe just add a comment here that this is the proof of the latest leaves per tree.